Add option to align add action on a repeater (#14431)
filamentphp/filament: packages/forms/docs/03-fields/12-repeater.md

    ->addActionLabel('Add member')
```

> ## Align the add action button

Reviewer: ```suggestion
### Aligning the add action button
```

> By default, the add action is align in the center. You may adjust this using the `addActionAlignment()` method:

Reviewer: ```suggestion
By default, the add action is aligned in the center. You may adjust this using the `addActionAlignment()` method, passing an `Alignment` option of `Alignment::Start` or `Alignment::End`:
```

```php
use Filament\Forms\Components\Repeater;
use Filament\Support\Enums\Alignment;

Repeater::make('members')
    ->schema([
        // ...
    ])

>     ->addActionAlignment(Alignment::Left)

Reviewer: ```suggestion
    ->addActionAlignment(Alignment::Start)
```
